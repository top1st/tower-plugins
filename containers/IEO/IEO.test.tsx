import { shallow } from 'enzyme';
import * as React from 'react';
import { IEO } from './index';

describe('IEO test', () => {
    it('should render', () => {
        const wrapper = shallow(<IEO />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toBeDefined();
    });
});
