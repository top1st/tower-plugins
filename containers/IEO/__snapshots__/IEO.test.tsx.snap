// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IEO test should render 1`] = `ShallowWrapper {}`;
